newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -211,11 +212,10 @@ export default function SummaryPage() {
           {isCreator && (
             <div className={styles['button-row']}>
               {gridViewActive && actions}
-              <Button
-                color="red"
-                className={styles['delete-button']}
```

Comment:
Was this removed on purpose?

Reply:
I just removed the class (which doesn't exist)